uint16 num_x
uint16 num_y
float32 spacing_x
float32 spacing_y
uint8 pattern
uint8 flags

# Specify how many times we want to upsample the image.
#  This is often useful for detecting small checkerboards far away
float32 width_scaling
float32 height_scaling

# Configure openCV's subpixel corner detector
uint32 subpixel_window
int32  subpixel_zero_zone

uint8 CHECKERBOARD = 0
uint8 CIRCLES      = 1
uint8 ACIRCLES     = 2
---
---
